### DISCUSSION TITLE
SmartAutoClicker-1.5.2

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#166] [Announcements]
# Change log:

This small release is focused on stability and performances:
* Crashes at detection stop should no longer occurs.
* Detection algorithm uses less memory and CPU.

# Note on background stability:

A lot of Android manufacturer prefer battery life over proper functionality of your apps. This might affect Smart AutoClicker stability on your device, and there is nothing I can do about it as an application developer.

However, you can find tips and tricks to keep Smart AutoClicker alive on your device here: https://dontkillmyapp.com

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/1.5.2'>SmartAutoClicker-1.5.2</a>.</em>

**Comment by yayabuzz** [comment]
I found same app in Google store call autoclicker pro is that your apps?

**Reply by Nain57** [reply]
No it is not. 

Smart AutoClicker is not available on the Google Play Store, only on [F-Droid](https://f-droid.org/packages/com.buzbuz.smartautoclicker/) or by apk direct download on Github below the selected [release](https://github.com/Nain57/Smart-AutoClicker/releases)

For the github apk, just in case you have trust issues:
* the build is made on github with the whole [source](https://github.com/Nain57/Smart-AutoClicker/blob/master/.github/workflows/release.yml) (code & build) available
* opencv dependency is build from the [source](https://github.com/Nain57/Smart-AutoClicker/blob/master/detection/fetchSources.gradle), no shady .so
* apk is automatically [build](https://github.com/Nain57/Smart-AutoClicker/actions/runs/2956394736) and posted when a release post is created
* those are signed correctly and can be updated manually

PS: can you provide me a link to this application ? Just to check if it's not a scam based on my app ;)

**Reply by yayabuzz** [reply]
https://play.google.com/store/apps/details?id=com.gameclicker.autoclicker.pro&hl=ar&gl=US&referrer=utm_source%3Dgoogle%26utm_medium%3Dorganic%26utm_term%3Dautoclicker+pro&pcampaignid=APPU_1_JPMRY_vAOL-YptQPlrWJ4Ao

**Reply by yayabuzz** [reply]
Can you create a password lock version for the scenario?

**Reply by Nain57** [reply]
This application is a poor copy of an old version.
I highly recommend to avoid it as it might be collecting sensitive data.

No password lock planned as it is not in the main scope of this application.

**Comment by RaufSab** [comment]
Keep it up bro, Your clicker is far superior than auto clicker pro (on Google store).

Is it challenging for you if I asked you whether you can add a feature which will allow the clicker to interact with a delivery app and make the clicker activate based either on the gps location of the customer (distance between the customer and the location of the driver who uses the delivery app), or make the clicker activate based on the region zip code the order coming from.

**Reply by RaufSab** [reply]
Below is the link to the delivery app I'm talking about if it will help you to figure out the requirements to develop the feature for your Smart auto clicker.

**Reply by RaufSab** [reply]
https://www.google.com/amp/s/m.apkpure.com/saned-j-driver/net.jahez.fleets/amp

**Comment by RaufSab** [comment]
Hi again Nain57,
Can you add a feature where you can define the number of clicks  and the number of swipes when the Smart clicker is activated?
Thanks ✌🏼

**Comment by RaufSab** [comment]
Hi Nain57,
Kindly if you have a Fiverr account let me know so that I can dm you in regards to the feature I'm asking for.

Have a great day.
